Library/Homebrew/utils.rb

```diff
@@ -121,6 +121,10 @@ def self.git_last_commit
     HOMEBREW_REPOSITORY.cd { `git show -s --format="%cr" HEAD 2>/dev/null`.chuzzle }
   end
 
+  def self.git_last_commit_date
+    HOMEBREW_REPOSITORY.cd { `git show -s --format="%cd" --date=short HEAD 2>/dev/null`.chuzzle }
+  end
```

Comment:
Don’t we already have that with `git_last_commit`?

```
$ brew irb
==> Interactive Homebrew Shell
Example commands available with: brew irb --examples
>> Homebrew.git_last_commit
=> "2 hours ago"
>>
```

This is used for the `config` command output.